org.freedesktop.Sdk.Extension.texlive.yml

```diff
@@ -75,6 +75,48 @@ modules:
           url: https://pi.kwarc.info/historic/systems/texlive/2021/
           version-pattern: texlive-([\d\.]+).source.tar.xz
           url-template: https://pi.kwarc.info/historic/systems/texlive/2021/texlive-$version-tlpdb-full.tar.gz
+  - name: perl
+    no-autogen: true
+    config-opts:
+      --des
+      --Duseshrplib
```

Comment:
```suggestion
      - --des
      - --Duseshrplib
```

This cause 
```
(flatpak-builder:2097273): Json-WARNING **: 18:36:35.977: Failed to deserialize "config-opts" property of type "GStrv" for an object of type "BuilderModule"
```

Reply:
but then with this change:
```
This GNU configure front end does not understand --des
```

Reply:
Nevermind updated the suggestion.

Reply:
@hfiguiere Thank you for your support. It really is a big help for me.